src/github_review_prompts/github_client.py

```
@@ -547,6 +743,179 @@ def get_resolved_comments_via_graphql(
+                    )
+
+                    # Outside diff range commentsを抽出
+                    if review_body and "Outside diff range comments" in review_body:
+                        try:
+                            from .utils.parsers import extract_outside_diff_comments
+
+                            extracted_outside = extract_outside_diff_comments(
+                                review_body
+                            )
+                        except ImportError as e:
+                            logger.error(f"Outside diff parser import failed: {e}")
```

Ruff (0.12.2)

Undefined name `logger`

(F821)